src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java

```diff
@@ -110,6 +114,10 @@ class FileColumnProperties {
     this.setMinStrNonCollated(null);
     this.setNullCount(stats.getCurrentNullCount());
     this.setDistinctValues(stats.getDistinctValues());
```

Comment:
we're not doing the if check for NDV, curious why we need to do it for NV?

Reply:
We already have NDV fields in FDN mode and it's always sets to -1, it's safe serialized it. NV is a new fields so adding this check to avoid it populated in FDN mode.